mysql-test/suite/tokudb/t/cluster_key_part.test

```diff
@@ -5,19 +5,25 @@ source include/have_partition.inc;
 set default_storage_engine='tokudb';
```

Comment:
IMHO it needs to be tested not only that DDL goes through, but that clustered indexes can actually be updated and data read back from them successfully. I have doubts whether that's in the scope of the current MP (do unpartitioned TokuDB tables have such tests?), but it has to be at least tried somehow before we ship the feature.